xonsh/environ.py

```diff
@@ -45,6 +40,8 @@
     seq_to_upper_pathsep,
 )
 
+from xonsh.prompt.base import FORMATTER_DICT
```

Comment:
You should not do `from ... import ...` for elements outside of the current package as this will not be lazy.